build/scripts/generate-docs.ps1

```diff
@@ -1,11 +1,11 @@
 param(
-    [string]$branch = "master",
+    [string]$branch = "main",
```

Comment:
default branch name on the docs repository is still `master`, please revert this